### DISCUSSION TITLE
Hello, is this an open source engine? It's free? Can it be used for the production of commercial games or software?

### DISCUSSION THREAD

**Discussion by RIKDXHQ11** [NVIDIA-Omniverse/PhysX#119] [General]
Hello, is this an open source engine? It's free? Can it be used for the production of commercial games or software?
Because I heard that it has become private again recently, I am very worried about it, so I came here to ask. thank you

**Comment by preist-nvidia** [comment]
Hi @RIKDXHQ11 - see the license here: https://github.com/NVIDIA-Omniverse/PhysX/blob/release/104.2/LICENSE.md

**Reply by RIKDXHQ11** [reply]
Will you continue to modify the license in the future? If its authorization method is unstable, I may consider refusing to use it.
